web/src/beta/features/Editor/DataSourceManager/index.tsx

```diff
@@ -37,6 +39,16 @@ const DataSourceManager: React.FC<DataProps> = ({ sceneId, onClose, onSubmit })
           id: "threeDTiles",
           label: "3D Tiles",
         },
+        {
+          content: <WmsTiles sceneId={sceneId} onSubmit={onSubmit} onClose={onClose} />,
+          id: "wms",
+          label: "WMS",
```

Comment:
Can you wrap all the labels with i18n? 🙏🏻 Most will be the same in Japanese, but some will be different.